### PR TITLE
Bit more Explanation for MultiEncoder

### LINKED ISSUE #2145: MultiEncoder needs more Explanation

### PR DESCRIPTION
**Pull request by passiweinberger**
fixes #2145

--------
EDIT: "fixes" should be here

### CODE REVIEW

File: nupic/encoders/multi.py [hunk 1]
```diff
@@ -147,6 +148,11 @@ def addMultipleEncoders(self, fieldEncodings):
                             name='consumption', minval=0,maxval=110,
                             clipInput=True, w=5, resolution=5),
     }
+    
+    would yield a SDR with a part encoded by the DateEncoder, 
+    and to parts seperately taken care of by the ScalarEncoder with the specified parameters. 
+    The three seperate encodings are then merged together to the final SDR.
```
Comment (breznak) [inline nupic/encoders/multi.py]: naming nit: encoders do not produce SDRs but "encodings/vectors", SDR is created in SP.

**Comment by breznak** [comment]
@passiweinberger @rhyolight wants an issue for each PR, and the PR should state in its description 
"fixes #xyz" 

:+1:

**Comment by passiweinberger** [comment]
fixes #2145

**Comment by breznak** [comment]
:+1: thanks!

**Comment by rhyolight** [comment]
Not sure why this is failing CI. The test failures don't make sense for these changes. 

@chetan51 Please review this PR.

File: nupic/encoders/multi.py [hunk 2]
```diff
@@ -104,6 +104,7 @@ def getDescription(self):
 
   ############################################################################
   def getWidth(self):
+        """Represents the sum of the widths of each fields encoding."""
```
Comment (chetan51) [inline nupic/encoders/multi.py]: Align indentation

**Comment by breznak** [comment]
> Not sure why this is failing CI. The test failures don't make sense for these changes.

some recent changes in NAB?
https://travis-ci.org/numenta/nupic/jobs/63229392#L2842

File: nupic/encoders/multi.py [hunk 3]
```diff
@@ -147,6 +148,11 @@ def addMultipleEncoders(self, fieldEncodings):
                             name='consumption', minval=0,maxval=110,
                             clipInput=True, w=5, resolution=5),
     }
+    
+    would yield a vector with a part encoded by the DateEncoder, 
+    and to parts seperately taken care of by the ScalarEncoder with the specified parameters. 
+    The three seperate encodings are then merged together to the final vector.
+    Note that these parts are always the same for each field.
```
Comment (chetan51) [inline nupic/encoders/multi.py]: I don't understand the last line, can you clarify?

**Comment by passiweinberger** [comment]
Better ?

**Comment by breznak** [comment]
> Better ?

close, but not yet :wink: 

the indent should be 2 spaces from method's `def`, as in
```
  def addMultipleEncoders(self, fieldEncodings):
    """
    fieldEncodings -- a dict of dicts, mapping field names to the field params
```

protip: run `pylint` on your file and @chetan51 will love you :smile:

**Comment by passiweinberger** [comment]
Closer? :D 
Oh nice Thanks :) This is what Spyder (the IDE i use) also uses, but it did not raise anything here :S

**Comment by breznak** [comment]
optimal :)

**Comment by chetan51** [comment]
:+1: Thanks!

**Pull request merged**